Testing updated front matter for shortguides

wildmanonline opened the pull request.

nmelehan commented:
:heavy_check_mark: Deploy Preview for *nostalgic-ptolemy-b01ab8* ready!


:hammer: Explore the source changes: 49a850936160cb183d0f2ca7c8ce2d4756147313

:mag: Inspect the deploy log: [https://app.netlify.com/sites/nostalgic-ptolemy-b01ab8/deploys/60af96233c432900095a393a](https://app.netlify.com/sites/nostalgic-ptolemy-b01ab8/deploys/60af96233c432900095a393a?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-4363--nostalgic-ptolemy-b01ab8.netlify.app/docs/whatsnew](https://deploy-preview-4363--nostalgic-ptolemy-b01ab8.netlify.app/docs/whatsnew?utm_source=github&utm_campaign=bot_dp)

wildmanonline commented:
This PR should be good to go. I moved all of the shortguides referenced in product documentation to their respective products folder under a `_shortguides` folder. Removed any unneeded front matter.
